plugins/source/alicloud/client/resolvers.go

```diff
@@ -0,0 +1,20 @@
+package client
+
+import (
+	"context"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/thoas/go-funk"
+	"time"
+)
+
+func ResolveParentColumn(field string) schema.ColumnResolver {
+	return func(_ context.Context, _ schema.ClientMeta, r *schema.Resource, c schema.Column) error {
+		return r.Set(c.Name, funk.Get(r.Parent.Item, field))
+	}
+}
+
+func ResolveUpdateDate() schema.ColumnResolver {
+	return func(_ context.Context, _ schema.ClientMeta, r *schema.Resource, c schema.Column) error {
+		return r.Set(c.Name, time.Now().AddDate(0, 0, -1).Format("2006-01-02"))
```

Comment:
Why is this column resolver needed, and why does it add `-1` days? Do you need information about exactly when a row was added? The `_cq_sync_time` column doesn't give exact row-level data, but it does tell you when a sync started, which is usually good enough.

Reply:
Oh, after reading a bit more of the PR, I see that this column is part of the PK, probably to avoid overwriting rows from previous syncs. You can also achieve the same by leaving the PK empty, in which case it will default to `_cq_sync_id`. In `overwrite-delete-stale` mode CloudQuery will first sync all rows, then delete rows from previous syncs.